dist/src/main/java/io/camunda/zeebe/broker/BrokerModuleConfiguration.java

```diff
@@ -38,7 +38,8 @@
     basePackages = {
       "io.camunda.zeebe.broker",
       "io.camunda.zeebe.shared",
-      "io.camunda.authentication"
+      "io.camunda.authentication",
+      "io.camunda.application.commons.metrics"
```

Comment:
why this package needs to be added in `BrokerModuleConfiguraiton` ? it is already scanned by [`CommonsModuleConfiguration`](https://github.com/camunda/camunda/blob/main/dist/src/main/java/io/camunda/application/commons/CommonsModuleConfiguration.java)